lib/public/views/Runs/format/formatRunDetectorsInline.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+import { RunDetectorQualities } from '../../../domain/enums/RunDetectorQualities.js';
+
+/**
+ * Format a list of detectors inline
+ *
+ * @param {Run} run the run for which detectors must be formatted
+ * @return {Component} the formatted run detectors inline
+ */
+export const formatRunDetectorsInline = (run) => {
```

Comment:
I believe here you can simply expect only the `detectorQualities` and `nDetectors` instead of the whole run and update the documentation of the param to `RunDetectorQuality[]` from `Run` similar to `formatRunDetectors.js`